yt_dlp/extractor/rumble.py

```diff
@@ -92,18 +175,28 @@ def _real_extract(self, url):
         }
 
         author = video.get('author') or {}
+        thumbnails = [{key: mapping[t_key] for t_key, key in self.THUMBNAIL_MAPPING.items()
+                       if t_key in mapping} for mapping in video.get('t', ())]
+        if not thumbnails and 'i' in video:
+            thumbnails.append({'url': video['i']})
+
+        if live_status in {'is_live', 'post_live'}:
+            duration = None
```

Comment:
what will `video['duration']` be in this case?

Reply:
Often events will have a preview / announcement / teaser video of about one minute. So `video['duration']` will be the length of that teaser.